CleanRepo/Program.cs

```diff
@@ -255,8 +255,8 @@ private static void ListOrphanedIncludes(string inputDirectory, Dictionary<strin
 
             Dictionary<string, int> includeFiles = new Dictionary<string, int>();
 
-            if (String.Compare(dir.Name, "includes", true) == 0
-                || String.Compare(dir.Name, "_shared", true) == 0)
+            if (string.Compare(dir.Name, "includes", true) == 0
```

Comment:
It's not better, in fact it is [functionally equivalent](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#string-vs-systemstring). For me it's a preference, and it's a matter of consistency. For example, would you say `Int32` or simply use the C# alias `int`? The same is true for `String` vs `string`. I tend to always use the provided aliases. It also can potential prevent a `using System;` too... which would certainly be nice.

| Alias | `System.*` |
| --: | :-- |
| `string` | `System.String` |
| `int` | `System.Int32` |
| `long` | `System.Int64` |
| `byte` | `System.Int16` |
| `decimal` | `System.Decimal` |
| ... there are | several more ... |

Reply:
The use of `var` I'm generally okay. I'm also okay with using `string` instead of `String` when defining a variable, but to me it's weird to call a method on an alias instead of the name of a class. So could we leave those as-is, e.g. `String.Compare`?

Reply:
Of course, again just a matter of preference. Out of curiosity do you do `Int32.TryParse` or `int.TryParse`?

Reply:
In that case I would use Int32.TryParse. It's just weird to me to call a method on a lower-case type.